pymc3/variational/advi_minibatch.py

```diff
@@ -305,6 +437,17 @@ def advi_minibatch(vars=None, start=None, model=None, n=5000, n_mcsamples=1,
     -------
     ADVIFit
         Named tuple, which includes 'means', 'stds', and 'elbo_vals'.
+
+    References
+    ----------
+    - Kingma, D. P., & Welling, M. (2014).
+      Auto-Encoding Variational Bayes. stat, 1050, 1.
+    - Kucukelbir, A., Ranganath, R., Gelman, A., & Blei, D. (2015).
+      Automatic variational inference in Stan. In Advances in neural
+      information processing systems (pp. 568-576).
+    - Blundell, C., Cornebise, J., Kavukcuoglu, K., & Wierstra, D. (2015).
+      Weight Uncertainty in Neural Network. In Proceedings of the 32nd
+      International Conference on Machine Learning (ICML-15) (pp. 1613-1622).
     """
     theano.config.compute_test_value = 'ignore'
```

Comment:
I have not seen this decolator. Do I need to define it or can I import from another module?